src/modules/shared/components/ListNavigation/ListNavigation.types.ts

```diff
@@ -13,7 +13,7 @@ export interface ListNavigationItem {
 	id: string;
 	active?: boolean;
 	hasDivider?: boolean;
-	children?: ListNavigationItem[];
+	children?: ListNavigationItem[] | null;
```

Comment:
`null` doorgeven lijkt mij wat riskant zeker als er hier in de code geen checks op zitten en erover gemapped zal worden. Zou in principe ook moeten werken zonder `| null`